maven-exec-plugin-yarn-example/pom.xml

```diff
@@ -0,0 +1,147 @@
+                <version>3.1.0</version>
+                <configuration>
+                    <filesets>
+                        <fileset>
+                            <directory>build</directory>
+                        </fileset>
+                    </filesets>
+                </configuration>
+            </plugin>
+        </plugins>
+        <pluginManagement>
+            <plugins>
+                <!-- Make yarn build cacheable -->
+                <plugin>
+                    <groupId>com.gradle</groupId>
+                    <artifactId>gradle-enterprise-maven-extension</artifactId>
+                    <configuration>
+                        <gradleEnterprise>
+                            <plugins>
+                                <plugin>
+                                    <groupId>org.codehaus.mojo</groupId>
+                                    <artifactId>exec-maven-plugin</artifactId>
+                                    <executions>
+                                        <execution>
+                                            <id>yarn-build</id>
+                                            <inputs>
+                                                <fileSets>
+                                                    <fileSet>
+                                                        <name>sources</name>
+                                                        <paths>
+                                                            <path>${project.basedir}/src</path>
+                                                            <path>${project.basedir}/public</path>
+                                                        </paths>
+                                                    </fileSet>
+                                                    <fileSet>
+                                                        <name>build-files</name>
+                                                        <paths>
+                                                            <path>${project.basedir}/package.json</path>
+                                                            <path>${project.basedir}/yarn.lock</path>
+                                                        </paths>
+                                                    </fileSet>
+                                                </fileSets>
+                                                <properties>
+                                                    <property>
+                                                        <name>executable</name>
+                                                    </property>
+                                                    <propert>
+                                                        <name>arguments</name>
+                                                    </propert>
+                                                </properties>
+                                                <ignoredProperties>
```

Comment:
Not sure it's correct to ignore all properties that we currently don't use.

Reply:
Yeah, maybe it would be better to make them input properties as well. 🤔

Reply:
Okay, I'll do another round.